lrspectrum/parsers.py

```diff
@@ -75,7 +75,10 @@ def _parse_gaussian(logfile):
     for i, line in enumerate(open(logfile)):
         if 'Excited State' in line[1:14]:
             lsp = line.split()
-            results[lsp[4]] = float(lsp[8].lstrip('f='))
+            if not is2c:
+                results[lsp[4]] = float(lsp[8].lstrip('f='))
+            else:
+                results[lsp[3]] = float(lsp[7])
```

Comment:
Needs testing. (`test__parse_gaussian()` in `test_parsers.py`)
You will probably need to add another file to `lrspectrum/test/data/` similar to `example_methane.log`